python/tidy/servo_tidy/tidy.py

```diff
@@ -675,8 +692,10 @@ def scan(only_changed_files=False, progress=True):
     errors = collect_errors_for_files(files_to_check, checking_functions, line_checking_functions)
     # wpt lint checks
     wpt_lint_errors = check_wpt_lint_errors(get_wpt_files(only_changed_files, progress))
+    # check dependecy licenses
+    dep_license_errors = check_dep_license_errors(progress)
```

Comment:
Also, doesn't this consume time? Wouldn't it be better if we check this when we run `./mach test-tidy --all` (i.e.,) `only_changed_files=False`?